### DISCUSSION TITLE
Unable to submit developer pack application

### DISCUSSION THREAD

**Discussion by BPilot253** [community/community#43127] [GitHub Education]
Hello, after having my support ticket closed due to the system being deprecated, I am here to ask the same question.

After being declined for the document I presented not being in English, I'm trying to submit a new application with its translation (that's by Google Lens). Unfortunately, I keep getting this error:

`
Your application cannot be reviewed until you fix the following: Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?,The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.,You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.,You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
`

I **am** a distant learner outside my school's country and the proof I provided states so. It has the school's name, address, date, my name, my grade etc. What should I do?

I hope someone can offer a solution to this non-sense...

**Comment by SilasRodrigues19** [comment]
Hi @BPilot253 

There are some discussions on this topic that have been resolved. Here are some links that may help you.


#38918
#30138
#37159

**Comment by jfullstackdev** [comment]
posting it here might help, as it can accelerate the request but there is no assurance, I'm seeing lots of similar posts like that, and not all are being responded
